cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -372,6 +372,7 @@ def start_kube_dns():
     }
 
     try:
+        create_addon('kubedns-cm.yaml', context)
```

Comment:
Seems like some of the scripts haven't been updated to create kube-dns service account. I will update them as well.